### DISCUSSION TITLE
Empty path when file picked from clicked file picker, but not when dropped?

### DISCUSSION THREAD

**Discussion by dreadylocks** [react-dropzone/react-dropzone#1216] [Q&A]
I'm posting here first in case this is user error and not a bug. Basically I've got the example code of the hook wrapper running in an electron app.

```typescript
const UploadWidget = () => {
  const [fileList, setFileList] = useState<File[]>([]);

  const handleDrop = (files: File[]) => {
    setFileList(files);
  };

  return (
    <Container>
      <FileInput style={{ marginBottom: 16 }} text={'Choose destination...'} />
      {fileList.length === 0 && (
        <Dropzone onDrop={handleDrop}>
          {({ getRootProps, getInputProps }) => (
            <StyledCard {...getRootProps()}>
              {/* @ts-ignore */}
              <FileInput {...getInputProps()} />
              <p>Drag and drop some files here, or click to select files</p>
            </StyledCard>
          )}
        </Dropzone>
      )}
      <FileUploadListContainer files={fileList} />
    </Container>
  );
};
```

When I drag a file into the zone, handleDrop receives the files the way you expect and I can inspect that each file has the full absolute path in the `path` attribute. However, the same file, if I click on the drop zone to open the filer picker, handleDrop shows that the files it passes have just the file name for `name` and an empty string for `path`. 

Am I missing something in the settings?

**Comment by sinecharta** [comment]
When you drag a file into an Electron app using react-dropzone, you gain access to the file's path because Electron operates within a Node.js context, which has access to the file system. However, when you select a file using the file selection dialog (click to select), the default behavior aligns with that of a web context, where, for security reasons, the file's path is not exposed.

**Comment by rolandjitsu** [comment]
@codelocksdev I'm not that familiar with electron, so I cannot say. I do see `FileInput` is being used, does that translate to a native `<input>`?

I remember a previous issue related to paths in electron, https://github.com/react-dropzone/file-selector/issues/36. It could be related to that.
